From 7a4601a7597220b3bd4ce5deffd58892e749d0e5 Mon Sep 17 00:00:00 2001
From: Yaoming Qin <b10945@freescale.com>
Date: Wed, 8 Sep 2010 16:18:49 +0800
Subject: [PATCH 075/502] ENGR00127282 The captured video clips have the a/v
 sync issue.

The root cause is that the input buffer for alsa is too small,
which will have the overrun issue.

Signed-off-by: Yaoming Qin <b10945@freescale.com>
---
 modules/alsa/alsa_imx5x.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/alsa/alsa_imx5x.cpp b/modules/alsa/alsa_imx5x.cpp
index 1a740ba..05d6a26 100755
--- a/modules/alsa/alsa_imx5x.cpp
+++ b/modules/alsa/alsa_imx5x.cpp
@@ -136,7 +136,7 @@ static alsa_handle_t _defaults[] = {
         channels    : 2,
         sampleRate  : DEFAULT_SAMPLE_RATE,
         latency     : 250000, // Desired Delay in usec
-        bufferSize  : 2048, // Desired Number of samples
+        bufferSize  : 6144, // Desired Number of samples
         modPrivate  : (void *)&setDefaultControls,
     },
 };
-- 
1.8.0

